src/Controller/UserController.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace BiffBangPow\MessageBoard\Controller;
+
+use BiffBangPow\MessageBoard\FormHandler\UserFormHandler;
+use Doctrine\ORM\EntityRepository;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use BiffBangPow\MessageBoard\Services\SessionService;
+
+class UserController
+{
+    /**
+     * @var \Twig_Environment
+     */
+    private $twig;
+
+    /**
+     * @var EntityRepository
+     */
+    private $userRepository;
+    /**
+     * @var UserFormHandler
+     */
+    private $userFormHandler;
+    /**
+     * @var SessionService
+     */
+    private $sessionService;
+
+    /**
+     * Controller constructor.
+     * @param \Twig_Environment $twig
+     * @param EntityRepository $userRepository
+     * @param UserFormHandler $userFormHandler
+     * @internal param EntityRepository $threadRepository
+     */
+    public function __construct(\Twig_Environment $twig, EntityRepository $userRepository, UserFormHandler $userFormHandler,SessionService $sessionService)
+    {
+        $this->twig = $twig;
+        $this->userRepository = $userRepository;
+        $this->userFormHandler = $userFormHandler;
+        $this->sessionService = $sessionService;
+    }
+
+    /**
+     * @param Request $request
+     * @return Response
+     */
+    public function getLoginAction(Request $request)
+    {
+        return new Response($content = $this->twig->render('login.html.twig', []));
```

Comment:
you don't need to have $content. All this is doing is assigning it to a variable that is never used in this scope which also returns to the value to the function. You can just do `return new Response($this->twig->render('login.html.twig', []));`